lib/combine_pdf/parser.rb

```diff
@@ -243,22 +243,22 @@ def _parse_
         ##########################################
         elsif str = @scanner.scan(/\<[0-9a-fA-F]*\>/)
           # warn "Found a hex string"
-          str = str.slice(1..-2).force_encoding(Encoding::ASCII_8BIT)
+          str = str.slice(1..-2).dup.force_encoding(Encoding::ASCII_8BIT)
```

Comment:
This dup is redundant and will use up a bunch of memory. The dynamic String returned from `String#slice` should be mutable.